app/models/role.rb

```diff
@@ -171,6 +182,7 @@ def clone(role_params = {})
   end
 
   def locked?
+    return false if self.class.modify_locked
```

Comment:
If I move it up here, I should be able to remove it in the validator (it's redundant) - I will not, because it's meant for 1.15 though